Jenkins.properties

```diff
@@ -1,22 +1,22 @@
 project.name=intyg
 
 artifact.name=logsender
-artifact.version=6.14.0.9
+artifact.version=7.0.0.0
 
 git.url=https://github.com/sklintyg/logsender.git
 git.branch=main
 
 builder.image=app-builder-image
-builder.image.tag=11.0.14
+builder.image.tag=21.0.2
 
 runtime.image=tomcat-base
-runtime.image.tag=9.0.78.5
+runtime.image.tag=10.1.30.1
 
-dependencies.infra.version=3.21.0.+
-dependencies.common.version=3.21.0.+
+dependencies.infra.version=4.0.0.+
+dependencies.common.version=4.0.0.+
 
 dependencies.infra.version.resolved=3.21.0.45
 dependencies.common.version.resolved=3.21.0.48
```

Comment:
should this be changed to 4.0.X also?

Reply:
Like 4.0.0.7
https://nexus.drift.inera.se/#browse/search=keyword%3Dse.inera.intyg.infra:NX.coreui.model.Component-1201